docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: load_data_dev
+
+___
+
+
+    Or to replace the existing task with a flow as the fourth step of the ``dev_org`` flow, first set the task to ``None`` and then insert the new flow.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: None
+                flow: my_flow
+
+You can swap two steps in a flow by replacing one with the other. If the steps are of different types (task/flow), the types being replaced first need to be set to ``None``.
+
+
+Configure Options on Tasks When Running a Subflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify options on tasks in subflows with the following syntax:
+
+.. code-block:: yaml
+
+    <flow_to_modify>:
+        steps:
+            <step_number>:
+                ``flow``: ``<sub_flow_name>``
+                options:
+                    ``<task>``:
+                        ``<option_name>``: ``<value>``
+
+Replace all values with ``<>`` with the desired values.
+
+    Example: Let's examine the definition of the ``ci_master`` flow from the universal ``cumulusci.yml`` file:
+
+.. code-block::
+
+    ci_master:
+        group: Continuous Integration
+        description: Deploy the package metadata to the packaging org and prepare for managed package version upload.  Intended for use against main branch commits.
+        steps:
+            1:
+                flow: dependencies
+                options:
+                    update_dependencies:
+                        include_beta: False
+            2:
+                flow: deploy_packaging
+            3:
+                flow: config_packaging
+
+This flow specifies that when the subflow ``dependencies`` runs, the ``include_beta`` option is passed a value of ``False`` to the ``update_dependencies`` task (which itself executes in the ``dependencies`` subflow). 
+
+
+``when`` Clauses
+^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a ``when`` clause in a flow step to conditionally run that step. A ``when`` clause is written in a Pythonic syntax that can evaluate to a boolean (``True`` or ``False``) result. 
+
+The variables that are available for reference in ``when`` clauses [TO BE ADDED]
+
+A common use case is to be able to check [TO BE ADDED]
+
+See `use variables for task options`_ for more information.
+
+
+
+Scratch Org Configurations
+--------------------------
+
+This section defines the scratch org configurations that are available without explicitly running ``cci org scratch`` to create a new configuration. If you're looking for more information on the use of scratch orgs with CumulusCI, see `Scratch Org Environments<TODO>`_. 
+
+
+Override Default Values
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: These overrides pertain only to scratch orgs.
+
+You can override these values for your org:
```

Comment:
```suggestion
You can override these values for your org.
```
Repeat style offender. "Introduce a vertical list with the beginning of a sentence followed by a colon. If the introduction is a complete sentence, end it with a period." https://confluence.internal.salesforce.com/display/DOCTEAM/lists

Reply:
I realllllly dislike this rule. Made the change.